scipy/fftpack/tests/test_basic.py

```diff
@@ -324,6 +324,29 @@ def test_invalid_sizes(self):
         assert_raises(ValueError, rfft, [])
         assert_raises(ValueError, rfft, [[1,1],[2,2]], -5)
 
+    # See Issue #5790
```

Comment:
I guess many previous references to Trac were of form "ticket #xxxx", but `gh-` could be better to make it a bit more explicit.